model/task/task_test.go

```diff
@@ -3901,6 +3901,53 @@ func TestByExecutionTasksAndMaxExecution(t *testing.T) {
 		assertTasksAreEqual(t, t1, tasks[0], 1)
 		assertTasksAreEqual(t, t2, tasks[1], 1)
 	})
+	t.Run("Fetching old executions when there are even older executions", func(t *testing.T) {
+		require.NoError(t, db.ClearCollections(Collection, OldCollection))
+
+		// Both tasks have two previous executions.
+		t1 := Task{
+			Id:        "t1",
+			Version:   "v1",
+			Execution: 2,
+			Status:    evergreen.TaskSucceeded,
+		}
+		assert.NoError(t, db.Insert(Collection, t1))
+
+		ot1_execution1 := t1
```

Comment:
We don't use snake case. Can all of these variables be modified to use camelcase?